usr.sbin/automount/automount.sh

```diff
@@ -0,0 +1,765 @@
+
+RETRY_COUNT (set to 3 by default)
+  How many times try to get file system type or try to mount.
+
+  example: RETRY_COUNT='1'
+
+RETRY_DELAY (set to 1 second by default)
+  Delay beetwin retry attempt.
+
+  example: RETRY_DELAY='2.5'
+
+USERUMOUNT (set to NO by default)
+  When set to YES it will 'chmod +s /sbin/umount'
+  which would allow an USER to unmount the file
+  system with their selected file manager.
+
+  example: USERUMOUNT='YES'
+
+NOTIFY (set to NO by default)
+  Use 'notify-send' and 'libnotify' to show notifications
+  of mounting and unmounting devices on the desktop.
+
+  example: NOTIFY='YES'
+
+WALL (set to NO by default)
+  Use wall(1) to show notifications of mounting and
+  unmounting devices on terminals of logged in users.
+
+  example: WALL='YES'
+
+FM ('exo-open --launch FileManager' by default)
+  If set to file manager command, the mount will
+  launch the specified command after successful
+  mount. Works only if USER parameter is also set.
+
+  example: FM='nautilus --browser --no-desktop'
+
+BLACKLIST (unset by default)
+  The automount will ignore devices defined here.
+
+  example: BLACKLIST='da0 da3s1a'
+
+USER (root by default)
+  If set to some username, the mount command will
+  chown(1) the mount directory with the user and
+  its primary user group. If used with FM option
+  allows to launch the specified file manager after
+  a successful mount.
+
+  example: USER="vermaden"
+
+REMOVEDIRS (set to YES by default)
+  If set to YES the automount(8) will remove /media dir after unmount.
+
+  example: REMOVEDIRS=NO
+
+NICENAMES (set to NO by default)
+  If set to YES the device/filesystem label will be used for /media dir name.
+
+  example: NICENAMES=YES
+
+IGNORE_SYS_PARTS (set to NO by default)
+  If set to YES automount(8) will ignore system partitions like EFI or MSR.
+
+  example: IGNORE_SYS_PARTS=YES
+
+EOF
+  exit 0
+}
+
+# display version if needed
+if [ "${1}" = '--version' -o \
+     "${1}" = '-version'  -o \
+     "${1}" = 'version'   -o \
+     "${1}" = '-v' ]
+then
+  echo
+  echo "                    ___        /\                                ___             "
+  echo "                 __/  /_      /  \                              _\  \__          "
+  echo "    ____   _____/_   __/__   /  _/\   ___ ___   ____ ______   __\__   _\         "
+  echo "   /    \ /  /  //  //    \ /\_/   \ /   /   \ /    \\\  \  \ /    \\\  \        "
+  echo "  /  /  //  /  //  //  /  //        \\\  \  \  \\\  \  \\\  \  \\\  \  \\\  \_   "
+  echo "  \_____\\\____/ \__\\\____//__________\\\__\__\__\\\____/ \_____\\\__\__\\\___\ "
+  echo
+  echo "automount 1.7.9 2022/05/24"
+  exit 0
+fi
+
+# display help if needed
+if [ "${1}" = "-h"     -o \
+     "${1}" = "--h"    -o \
+     "${1}" = "-help"  -o \
+     "${1}" = "--help" -o \
+     "${#}" -eq "0"    -o \
+     "${#}" -eq "1" ]
+then
+  __usage
+fi
+
+# read configuration files
+if [ -f /etc/automount.conf ] ; then
+  . /etc/automount.conf
+fi
```

Comment:
This is the only difference between this automount and the upstream [vermaden/automount](https://github.com/vermaden/automount). vermaden/automount doesn't support `/etc/automount.conf` by default.